web/src/main/java/org/cbioportal/web/config/WebServletContextListener.java

```diff
@@ -30,6 +32,7 @@ public void contextInitialized(ServletContextEvent arg0) {
         
         this.showOncokb = Boolean.parseBoolean(getProperty("show.oncokb", "true"));
         this.oncokbToken = getProperty("oncokb.token", "");
+        this.oncokbURL = getProperty("oncokb.public_api.url", "");
```

Comment:
@zhx828 do you think we need to set a default url for this one?

Reply:
@leexgh I don't think so. We use its empty status for later printing out the warning message.